src/sentry/runner/commands/devservices.py

```diff
@@ -303,6 +303,19 @@ def up(
     """
     from sentry.runner import configure
 
+    click.secho(
+        """
+WARNING: We're transitioning from `sentry devservices` to the new and improved `devservices` in the near future.
```

Comment:
Should we give an estimate on how far into the future this is?

Reply:
Fair, should have people develop some sort of urgency as this will be done next month